package.json

```diff
@@ -15,16 +15,23 @@
     "clean": "rm -rf dist",
     "clean:all": "ts-node ./scripts/clean-all",
     "fix": "yarn lint:js --fix",
+    "kitchen": "yarn workspace @guardian/source-kitchen-react",
     "lint": "yarn lint:js && yarn lint:styles",
     "lint:js": "eslint . --ext .ts,.tsx,.js",
     "lint:styles": "stylelint src/core/**/styles.ts",
     "prepare": "husky install",
     "publish:release": "ts-node ./scripts/publish-release",
+    "publish:release:kitchen": "ts-node ./scripts/publish-release-kitchen",
     "release:major:all": "yarn build:all && yarn validate && yarn verbump:major:all && yarn publish:release",
+    "release:major:kitchen": "yarn kitchen build && yarn validate && yarn kitchen verbump:major && yarn publish:release:kitchen",
     "release:minor:all": "yarn build:all && yarn validate && yarn verbump:minor:all && yarn publish:release",
+    "release:minor:kitchen": "yarn kitchen build && yarn validate && yarn kitchen verbump:minor && yarn publish:release:kitchen",
     "release:patch:all": "yarn build:all && yarn validate && yarn verbump:patch:all && yarn publish:release",
+    "release:patch:kitchen": "yarn kitchen build && yarn validate && yarn kitchen verbump:patch && yarn publish:release:kitchen",
     "release:prerelease-bump:all": "yarn validate && yarn verbump:prerelease:all && yarn publish:release",
+    "release:prerelease-bump:kitchen": "yarn validate && yarn kitchen verbump:prerelease && yarn publish:release:kitchen",
     "release:prerelease:all": "yarn validate && yarn publish:release",
+    "release:prerelease:kitchen": "yarn validate && yarn publish:release:kitchen",
```

Comment:
This adds a release process for the kitchen which mirrors that of the rest of Source. For example, to release a new minor version, run `yarn release:minor:kitchen`. This will:

* build the kitchen
* run validation
* bump the version in the kitchen `package.json
* release the new version of the kitchen to npm

We could look at implementing something else (e.g. [np](https://github.com/sindresorhus/np)) but I thought that, for the sake of keeping the initial change as small as possible, it made sense to duplicate the existing process for now.

Reply:
LGTM 👍